app/views/application/_toasts.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="fixed top-24 w-full z-50">
+  <% if notice %>
+    <%= render 'application/toast',
+      message:       notice,
+      bg_color:      'bg-green-100',
+      close_classes: 'bg-green-100 hover:bg-green-200 text-green-500 focus:ring-offset-green-50 focus:ring-green-600',
+      icon:          'svg/check_circle',
+      icon_color:    'text-green-400',
+      text_color:    'text-green-800'
```

Comment:
This feels really leaky.

Reply:
It 100% is. I don't to deduce the toast DOM and originally was just passing in a color and having the toast create the classes. That led to tailwind not being able to detect the class usage and incorrectly purging classes. The only other option I thought of was to maintain both toasts and keep them in sync manually.

Reply:
Use a `application_helper` to generate a class for the flash message type then use CSS to style each one.

Reply:
well that's a plan and a half